HomeWork_3/Task_3.py

```diff
@@ -0,0 +1,38 @@
+'''Найти самое длинное слово в введенном предложении.
+Учтите что в предложении есть знаки препинания.'''
+line = str(input('Enter string \n'))
+list_words = line.split()
+LongWord = 0
+for i in range(1, len(list_words)):
+    if len(list_words[LongWord]) < len(list_words[i]):
+        LongWord = i
+LongWord = list_words[LongWord].strip(",")
+print(LongWord)
+
+'''Вводится строка. Требуется удалить из нее
+повторяющиеся символы и все пробелы. Например,
+если было введено "abc cde def", то должно быть
+выведено "abcdef".'''
+text = str(input('Enter string \n'))
+text = text.strip()
+text = text.split()
+text = ''.join(text)
+print(text)
```

Comment:
Your solution doesn't remove letters duplicates

Reply:
text = str(input('Enter string \n'))
text = text.split()
text = ''.join(text)
text_nondublicate = []
for i in text:
    if i not in text_nondublicate:
        text_nondublicate.append(i)
print(''.join(text_nondublicate))